docs/guides/es2k/enabling-comms-channel.md

```diff
@@ -0,0 +1,131 @@
+# Enabling the COMMS Channel
+
+The COMMs channel permits different compute complexes on the IPU to communicate with each other.
```

Comment:
This is refering to the communication between imc, acc and host right? maybe we should provide more info for user, for those that dont know the architecture

Reply:
The information related to compute complexes can't be part of opensource docs

Reply:
Not asking for detailed info, but clarifying what "_different compute complexes_" means. This statement seems to assume that user is familiar with details

Reply:
This is supposed to give the customer a rough idea of what COMMS means without going into detail. I don't see a need to explain what we mean by _Compute complexes_.